pkg/instance/instance.go

```diff
@@ -261,10 +261,10 @@ func (inst *inst) test() error {
 		}
 		if bootErr, ok := err.(vm.BootErrorer); ok {
 			testErr.Title, testErr.Output = bootErr.BootError()
-			rep := inst.reporter.Parse(testErr.Output)
+			rep := inst.reporter.Parse(testErr.Output, 0)
```

Comment:
I see lots of callers pass 0 here, maybe instead of changing Parse, we add ParseOffset that additionally accepts offset.